src/main/scripts/startMZmine_Windows.bat

```diff
@@ -1,12 +1,44 @@
 @echo off
 
-rem This is necessary to access the TOTAL_MEMORY and ADDRESS_WIDTH variables inside the IF block
+:: *****************************************
+:: Optional values - Please modify as needed
+:: *****************************************
+
+:: Total amount of memory in MB available to MZmine 2.
+:: 0 = automatically determined

```

Comment:
the comment says 0 = automatically determined, but below AUTO is used instead of 0.